src/utils/protostar_directory.py

```diff
@@ -46,24 +47,37 @@ def protostar_version(self) -> VersionType:
                 version_s = tomli.loads(file.read())["tool"]["poetry"]["version"]
                 return VersionManager.parse(version_s)
         except FileNotFoundError:
-            getLogger().warning("Couldn't read Protostar version.")
-            return VersionManager.parse("0.0.0")
+            getLogger().warning("Couldn't read Protostar version")
+            return None
 
     @property
-    def cairo_version(self) -> VersionType:
+    def cairo_version(self) -> Optional[VersionType]:
         path = (
             self._protostar_directory.directory_root_path
             / "dist"
             / "protostar"
             / "info"
             / "pyproject.toml"
         )
-        with open(path, "r", encoding="UTF-8") as file:
-            version_s = tomli.loads(file.read())["tool"]["poetry"]["dependencies"][
-                "cairo-lang"
-            ]
-            return VersionManager.parse(version_s)
+        try:
+            with open(path, "r", encoding="UTF-8") as file:
+                version_s = tomli.loads(file.read())["tool"]["poetry"]["dependencies"][
+                    "cairo-lang"
+                ]
+                return VersionManager.parse(version_s)
+        except FileNotFoundError:
+            getLogger().warning("Couldn't read cairo-lang version")
+            return None
+
+    @property
+    def git_version(self) -> Optional[VersionType]:
+        output = Git().execute(["git", "--version"])
+        # pylint: disable=unidiomatic-typecheck
+        if type(output) is str:
+            return version.parse(output.replace("git version ", ""))
+        return None
 
     def print_current_version(self) -> None:
-        print(f"Protostar version: {self.protostar_version}")
-        print(f"Cairo-lang version: {self.cairo_version}")
+        print(f"Protostar version: {self.protostar_version or 'unknown'}")
+        print(f"Cairo-lang version: {self.cairo_version or 'unknown'}")
+        print(f"Git version: {self.git_version or 'unknown'}")
```

Comment:
I think we should not include git here, git has it's own command to check version, it's not a responsibility of protostar.

Reply:
I made this change to print everything what is required by the updated issue template. I'll modify the issue template instead.